pennylane/tape/qscript.py

```diff
@@ -373,6 +373,61 @@ def op_wires(self) -> Wires:
 
     ##### Update METHODS ###############
 
+    def update(self, **kwargs):
+        r"""update(operations, measurements, shots, trainable_params)
+
+        Update attributes on the tape.
+
+        Keyword Args:
+            operations (Iterable[Operator]): An iterable of the operations to replace on the tape.
+            measurements (Iterable[MeasurementProcess]): An iterable of all the measurements to
+                replace on the tape.
+            shots (None, int, Sequence[int], ~.Shots): Number and/or batches of shots to replace
+                on the tape.
+            trainable_params (None, Sequence[int]): the indices for which parameters are trainable.
+
+        Returns: A new tape instance, initialized with any kwargs passed to update.
+            Anything not set in update will be the same as the initial tape.
+
+        Anything not set by update will retain the values on the original tape.
```

Comment:
```suggestion
        Returns: 
            A new tape instance initialized with any kwargs passed to update.
            
        Anything not set in update will be the same as the initial tape.
        Anything not set by update will retain the values on the original tape.
```

This is to avoid rendering issues (see attached pic)
![Screenshot from 2024-09-20 10-23-42](https://github.com/user-attachments/assets/7f3b501a-2df9-4998-baad-5ae89f586da5)

Reply:
Thanks! I've moved this up to be a kwarg in `copy` instead, so this part of the docstring is substantially different now.